apps/web/src/components/Menu/hooks/useTradingRewardStatus.ts

```diff
@@ -0,0 +1,20 @@
+import { useMemo } from 'react'
+import useAllTradingRewardPair from 'views/TradingReward/hooks/useAllTradingRewardPair'
+
+export const useTradingRewardStatus = () => {
+  const { data: allTradingRewardPairData } = useAllTradingRewardPair()
+  const latestCampaignId = allTradingRewardPairData.campaignIds[allTradingRewardPairData.campaignIds.length - 1]
+
+  return useMemo(() => {
+    const currentTime = new Date().getTime() / 1000
+    const incentive = allTradingRewardPairData.campaignIdsIncentive.find(
+      (i) => i.campaignId.toLowerCase() === latestCampaignId.toLowerCase(),
```

Comment:
better add optional mark to `latestCampaignId` if it's possibly undefined

Reply:
done